ui/components/providers/radio-group-provider.tsx

```diff
@@ -5,24 +5,82 @@ import React from "react";
 import { Control, Controller } from "react-hook-form";
 import { z } from "zod";
 
-import { addProviderFormSchema } from "@/types";
-
 import {
   AWSProviderBadge,
   AzureProviderBadge,
   GCPProviderBadge,
+  GitHubProviderBadge,
   KS8ProviderBadge,
   M365ProviderBadge,
-} from "../icons/providers-badge";
-import { CustomRadio } from "../ui/custom";
-import { FormMessage } from "../ui/form";
+} from "@/components/icons/providers-badge";
+import { CustomRadio } from "@/components/ui/custom";
+import { FormMessage } from "@/components/ui/form";
+import { addProviderFormSchema, IconSvgProps } from "@/types";
 
 interface RadioGroupProviderProps {
   control: Control<z.infer<typeof addProviderFormSchema>>;
   isInvalid: boolean;
   errorMessage?: string;
 }
 
+const PROVIDERS_CONFIG = [
```

Comment:
Is this included in the scope of this PR? Could you clarify the exact scope of this PR?